.circleci/config.yml

```diff
@@ -14,11 +14,11 @@ jobs:
     steps:
       - checkout
       - restore_cache:
-          key: v3-npm-{{ checksum "package-lock.json" }}
+          key: v4-npm-{{ checksum "package-lock.json" }}
```

Comment:
You added v5 in the listings project is this more of a version of the iteration up to invalidate the caches?

Reply:
yeah, it's just an incremental counter per-project. there's no meaning to it